app/services/terms_of_use/acceptor.rb

```diff
@@ -20,6 +20,12 @@ def perform!
 
     attr_reader :user_account, :version, :common_name
 
+    def validate_common_name(common_name:)
+      raise ArgumentError, '[TermsOfUse][Acceptor]: common_name not found' if common_name.blank?
```

Comment:
Can we use a custom error for this? (The reason I prefer custom errors is it lets us rescue this specific error, and not accidentally rescue other potential `ArgumentError` that we actually want to crash and go to Sentry, since they'd be unexpected and we'd want to investigate what happened)

Reply:
Also, don't want the `[TermsOfUse][Acceptor]` part for the error message, since this will be rendered and returned to the API caller